README.md

```diff
@@ -1,7 +1,7 @@
 # [GitHub Desktop](https://desktop.github.com)
 
-GitHub Desktop is an open source [Electron](https://www.electronjs.org/)-based
-GitHub app. It is written in [TypeScript](http://www.typescriptlang.org) and
+[GitHub Desktop](https://desktop.github.com/) is an open source [Electron](https://www.electronjs.org/)-based
+Official GitHub app. It is written in [TypeScript](http://www.typescriptlang.org) and
```

Comment:
Let's avoid the "official" part here. That should be evident by the name and I don't we don't need to be more direct than that.

```suggestion
GitHub app. It is written in [TypeScript](http://www.typescriptlang.org) and
```